man/bfcfg.8

```diff
@@ -0,0 +1,31 @@
+.TH BFCFG 8 "June 2020"
+.SH NAME
+bfcfg \- BlueField SoC configuration utilities
+.SH SYNOPSIS
+.B bfcfg
+.RB [ \-\-help | \-h ]
+.PP
+.B bfcfg
+.RB [ \-\-dump | \-d ]
+.PP
+.B bfcfg
+.RB [ \-\-part-info | \-p ]
+.SH DESCRIPTION
+BlueField devices may be configured by a series of configuration options
+contained in the
+.I /etc/bf.cfg
+file. The bf.cfg file is in a simple key-value pair format. For more
+information about the keys and their use, see the source code of
+.BR bfcfg .
+Running this script will update various aspects of the system to reflect the
+settings specified in bf.cfg.
+.SH OPTIONS
+.IP "--dump | -d"
+Instead of configuring the system, dump current configuration values.
+.IP "--part-info | -p"
+Configure and export partition information for install purposes, rather than
+performing a normal configuration. Used by
```

Comment:
Parse the partition configuration in /etc/bf.cfg and export variable $EFI_PART, $ROOT_PART and
$PERSIST_PART. Used by